Sources/Logging/Strings/ConfigureStrings.swift

```diff
@@ -37,6 +37,8 @@ enum ConfigureStrings {
 
     case response_verification_mode(Signing.ResponseVerificationMode)
 
+    case storekit_version(String)
```

Comment:
This `String` could be anything. Better to use `StoreKitVersion` to make it type safe.